ds_judgements_public_ui/static/markdown/court_descriptions/uksc.md

```diff
@@ -0,0 +1 @@
+The UK Supreme Court is the final court for appeals in civil and criminal cases from across the UK. Typically this court deals with important legal questions that affect the public or the constitution. The UK Supreme Court was established in October 2009. You can read more about the role of the court on the [United Kingdom Supreme Court](https://www.supremecourt.uk/){target="\_blank"} website.
```

Comment:
I was wondering whether it makes more sense for this to be a part of the utils data about courts; but it's fine here and I don't think we'll want to reuse it elsewhere.

Reply:
aha yep there was a discussion on slack about this a while back - i think we settled on 'something easily editable in the PUI' just to make the turnaround for edits a bit quicker - this way we don't have to do a utils PR, then a utils version bump PR, then a utils release, then a PUI PR, then a PUI deploy if we want to tweak the text.